examples/covariance/plot_outlier_detection.py

```diff
@@ -49,41 +49,42 @@
 classifiers = {
     "One-Class SVM": svm.OneClassSVM(nu=0.95 * outliers_fraction + 0.05,
                                      kernel="rbf", gamma=0.1),
-    "robust covariance estimator": EllipticEnvelope(contamination=.1),
-    "Isolation Forest": IsolationForest(max_samples=n_samples, random_state=rng)}
+    "Robust covariance": EllipticEnvelope(contamination=outliers_fraction),
+    "Isolation Forest": IsolationForest(max_samples=n_samples,
+                                        contamination=outliers_fraction,
+                                        random_state=rng)}
 
 # Compare given classifiers under given settings
 xx, yy = np.meshgrid(np.linspace(-7, 7, 500), np.linspace(-7, 7, 500))
 n_inliers = int((1. - outliers_fraction) * n_samples)
 n_outliers = int(outliers_fraction * n_samples)
 ground_truth = np.ones(n_samples, dtype=int)
-ground_truth[-n_outliers:] = 0
+ground_truth[-n_outliers:] = -1
 
 # Fit the problem with varying cluster separation
 for i, offset in enumerate(clusters_separation):
     np.random.seed(42)
     # Data generation
-    X1 = 0.3 * np.random.randn(0.5 * n_inliers, 2) - offset
-    X2 = 0.3 * np.random.randn(0.5 * n_inliers, 2) + offset
+    X1 = 0.3 * np.random.randn(n_inliers // 2, 2) - offset
+    X2 = 0.3 * np.random.randn(n_inliers // 2, 2) + offset
```

Comment:
Oh you want it rounded too. Nevermind. Sorry for the noise. Deleting this comment.